src/main/java/TaskManager.java

```diff
@@ -0,0 +1,90 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+    private ArrayList<Task> taskList;
+
+    public TaskManager() {
+        this.taskList = new ArrayList<Task>();
+    }
+
+    public void parseCommand(String command) throws DukeException {
```

Comment:
The parseCommand is very clean and neat!

Reply:
Thank you!